submodules/device/device.js

```diff
@@ -979,9 +979,10 @@ define(function(require) {
 				form_data.caller_id.internal.number = form_data.caller_id.internal.number.replace(/\s|\(|\)|-|\./g, '');
 				form_data.caller_id.external.number = form_data.caller_id.external.number.replace(/\s|\(|\)|-|\./g, '');
 				form_data.caller_id.emergency.number = form_data.caller_id.emergency.number.replace(/\s|\(|\)|-|\./g, '');
-				form_data.caller_id.asserted.number = _.isEmpty(form_data.caller_id.asserted.number)
-					? ''	// Need to keep the empty string so the new value is not lost when merging the form data with the original data
-					: monster.util.getFormatPhoneNumber(form_data.caller_id.asserted.number).e164Number;
+
+				if (!_.chain(form_data.caller_id).get('asserted.number').isEmpty().value()) {
```

Comment:
You should define a default value (matching the type of the expected one when possible) when using `lodash#get`, unless you are specifically checking against `undefined`, as we don't want our logic to rely on the inner workings of `lodash` too much (e.g. returning `true` on `_.isEmpty(undefined)`).